scripts/helpers/getMedGasPrice.js

```diff
@@ -0,0 +1,14 @@
+require("dotenv").config();
+const { BN } = require("@openzeppelin/test-helpers");
+const axios = require("axios").default;
+
+const API_URL = `https://api.etherscan.io/api?module=gastracker&action=gasoracle&apikey=${process.env.ETHERSCAN_API_KEY}`;
+
+module.exports = async function () {
```

Comment:
Could just extend this [function here](https://github.com/ribbon-finance/structured-products/blob/master/scripts/helpers/getGasPrice.js) to use the median gas price.